src/components/company/candidates/candidate-card.tsx

```
@@ -0,0 +1,650 @@
+
+  const visibleActions = actions.filter(a => !a.hidden?.(candidate));
+  const hasAIAnalysis = candidate.has_ai_analysis;
+
+  // Handle keyboard navigation in menu
+  const handleMenuKeyDown = (e: React.KeyboardEvent, index: number) => {
+    e.stopPropagation();
+    
+    switch (e.key) {
+      case 'ArrowDown':
+        e.preventDefault();
+        const nextIndex = index < visibleActions.length - 1 ? index + 1 : 0;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)